.ci/run_container.sh

```diff
@@ -12,7 +13,13 @@ then
   fi
 fi
 
-"${CONTAINER_RUNTIME}" run --rm --detach --name "pulp" --volume "${BASEPATH}/settings:/etc/pulp" --publish "8080:80" pulp/pulp-fedora31
+if [ -n "$PULPCORE_VERSION" ]
+then
+  "${CONTAINER_RUNTIME}" build -f "${BASEPATH}/Containerfile" -t pulp --build-arg PULPCORE_VERSION="==$PULPCORE_VERSION" .
+  IMAGE="pulp"
+fi
```

Comment:
It would be really nice to have those images available from pulp-oci-images

Reply:
Here we go: https://github.com/pulp/pulp-oci-images/pull/37